python/feathub/feature_views/tests/test_feature_view.py

```diff
@@ -78,7 +78,7 @@ def test_features(self):
         )
         built_feature_view = self.registry.build_features([feature_view_2])[0]
 
-        self.assertEqual(feature_1, built_feature_view.get_feature("feature_1"))
+        self.assertIsNotNone(built_feature_view.get_feature("feature_1"))
```

Comment:
If it is not equal `feature_1`, what would be the expected value? Can we specify the expected value here?